File: src/game/events/UpdateCurrentBombsLengthEvent.java
package game.events;

import game.Bomberman;
import game.entity.models.BomberEntity;

public class UpdateCurrentBombsLengthEvent implements GameEvent {
    @Override
    public void invoke(Object arg) {
        Bomberman.getMatch().getCurrentLevel().onUpdateBombsLengthEvent(Bomberman.getMatch().getPlayer(), (int) arg);
    }
}

File: src/game/events/game/UpdateCurrentBombsLengthEvent.kt
package game.events.game

import game.Bomberman
import game.events.models.GameEvent

class UpdateCurrentBombsLengthEvent : GameEvent {
    override fun invoke(arg: Any?) {
        Bomberman.getMatch().currentLevel!!.eventHandler.onUpdateBombsLengthEvent(Bomberman.getMatch().player ?: return, arg as Int)
    }
}